packages/sidecar/src/main.ts

```diff
@@ -205,11 +205,25 @@ function log(...args: any[]) {
 
 let serverInstance: Server;
 
-export function setupSidecar(): void {
+const isValidPort = (value: string) => {
+  const portNumber = parseInt(value, 10);
+  return /^\d+$/.test(value) && portNumber > 0 && portNumber <= 65535;
+};
+
+export function setupSidecar(port: string): void {
```

Comment:
Can we make this parameter optional and accept string | number? I get the string use case for CLI but we also call this in some frameworks (sveltekit, docs coming soon) and it feels unintuitive to pass the port as a string.

Reply:
Yup, makes sense. fixing it